configure.ac

```diff
@@ -63,8 +63,6 @@ AC_CONFIG_FILES([
 	etc/Makefile
 	etc/init.d/Makefile
 	etc/zfs/Makefile
-	etc/systemd/Makefile
-	etc/systemd/system/Makefile
```

Comment:
I don't think this belong in contrib.  This is code we support and maintain as part of the project so I'd like to leave where it is.